eox_core/settings/production.py

```diff
@@ -141,8 +145,8 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
             integrations=[
                 DjangoIntegration(),
             ],
-
             # If you wish to associate users to errors (assuming you are using
             # django.contrib.auth) you may enable sending PII data.
-            send_default_pii=True
+            send_default_pii=True,
+            **sentry_extra_options
```

Comment:
If I add a duplicated option (I did it by accident), instances won't be able to initialize. So can you please add a try-except for this?
![image](https://user-images.githubusercontent.com/64440265/225420042-305132b8-ff7f-47c3-b35d-f29d2d02b24a.png)

Reply:
Instances won't initialize either (at least workers) if the option doesn't exist:
![image](https://user-images.githubusercontent.com/64440265/225419491-38d69a3a-54f6-4fc1-b4c7-0a4a986215e9.png)

Reply:
I've added a try-except block that will ignore the `sentry_extra_options` param if sentry fails to initialize

Reply:
What do you think? @mariajgrimaldi

Reply:
Just a a reminder @mariajgrimaldi